src/planner/ompl/Planner.cpp

```diff
@@ -168,7 +170,15 @@ trajectory::InterpolatedPtr planOMPL(
     statespace::InterpolatorPtr _interpolator,
     double _maxPlanTime)
 {
-  _planner->setProblemDefinition(_pdef);
+  try
+  {
+    _planner->setProblemDefinition(_pdef);
+  }
+  catch(::ompl::Exception &ex)
+  {
+    std::cout << "Error" << std::endl;
```

Comment:
Don't print to `std::out`. if you want to return this error message, I suggest either wrapping it in an Aikido exception or putting it in a `PlannerResult` output parameter.